element_animal/injection.py

```diff
@@ -0,0 +1,159 @@
+import importlib
+import inspect
+
+import datajoint as dj
+
+from element_lab import lab
+from . import surgery
+
+schema = dj.Schema()
+_linking_module = None
+
+
+def activate(
+    injection_schema_name: str,
+    surgery_schema_name: str = None,
+    lab_schema_name: str = None,
+    *,
+    create_schema: bool = True,
+    create_tables: bool = True,
+    linking_module: bool = True,
+):
+    """Activate this schema.
+
+    Args:
+        schema_name (str): schema name on the database server to activate the
+                        `subject` element
+        create_schema (bool): when True (default), create schema in the
+                            database if it does not yet exist.
+        create_tables (bool): when True (default), create tables in the
+                            database if they do not yet exist.
+        linking_module (bool): a module name or a module containing the
+        required dependencies to activate the `subject` element:
+
+    Dependencies:
+    Upstream tables:
+        User: the who conducted a particular surgery/implantation
+    """
+
+    if isinstance(linking_module, str):
+        linking_module = importlib.import_module(linking_module)
+    assert inspect.ismodule(
+        linking_module
+    ), "The argument 'linking_module' must be a module's name or a module"
+
+    global _linking_module
+    _linking_module = linking_module
+
+    lab.activate(
+        lab_schema_name,
+        create_schema=create_schema,
+        create_tables=create_tables,
+    )
```

Comment:
On second thought, should we pass the `lab.Device` table into this namespace through the `linking_module`?  Similar to how we have done for other Elements (e.g. with `element-calcium-imaging`, the `Session` table is passed to the `scan` module through the `linking_module`).

Reply:
We could do that, but I assume we typically use the `linking_module` to link tables from two different Elements rather than tables from different modules? Is that correct?

Reply:
Exactly.  So we can pass the `lab.Device` table from `element-lab` into the `injection` schema of `element-animal`.

Reply:
Oh that's my mistake. For some reason, I thought this was within `element-animal`. And in this case, yes, I agree. I'll make that change in the next commit.

Reply:
Thanks!